_posts/en/newsletters/2024-08-16-newsletter.md

```diff
@@ -0,0 +1,324 @@
+  problems in the released code from affecting your configuration.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.1][] is a release candidate for this library for
+  building wallets and other Bitcoin-enabled applications.  The original
+  `bdk` Rust crate has been renamed to `bdk_wallet` and lower layer
+  modules have been extracted into their own crates, including
+  `bdk_chain`, `bdk_electrum`, `bdk_esplora`, and `bdk_bitcoind_rpc`.
+  The `bdk_wallet` crate "is the first version to offer a stable 1.0.0 API."
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+<!-- FIXME:Gustavojfe -->
+
+- [Bitcoin Core #29519][] resets the `pindexLastCommonBlock` value after an
+  [assumeUTXO][topic assumeutxo] snapshot is loaded, so that a node prioritizes
+  downloading blocks after the most recent block in the snapshot. This fixes a
+  bug where a node would set `pindexLastCommonBlock` from existing peers before
+  loading the snapshot and start downloading blocks from that block, defeating
+  the purpose of assumeUTXO.
+
+- [Bitcoin Core #30598][] removes block height from the [assumeUTXO][topic
+  assumeutxo] snapshot file metadata as it is not a unique identifier in a
+  pre-sanitized untrusted file compared to the block hash that is kept. A node
+  can still obtain block height from many other internal sources.
+
+- [Bitcoin Core #28280][] optimizes Initial Block Download (IBD) performance for
+  pruned nodes by not emptying the UTXO cache during pruning flushes. It does
+  this by listing "dirty" cache entries, entries that have changed since they
+  were last written to the database, separately. This allows a node to avoid
+  unnecessarily scanning the entire cache during prune flushes and instead focus
+  on the dirty entries.  The optimization results in up to 32% faster IBD for
+  pruned nodes with high `dbcache` settings, and about 9% improvement with
+  default settings. See Newsletter [#304][news304 cache].
+
+- [Bitcoin Core #28052][] adds XOR encoding to `blocksdir *.dat` files on
+  creation as a preventative mechanism against unintentional and accidental data
+  corruption by anti-virus software or other type of software. This can be
```

Comment:
```suggestion
  corruption by anti-virus or similar software. This can be
```
Or, if that's not quite the right meaning, probably should be "other types"